src/ucs/datastruct/piecewise_func.h

```diff
@@ -63,29 +64,29 @@ double ucs_piecewise_func_apply(const ucs_piecewise_func_t *func, size_t x);
 
 
 /**
- * Add the trend on the provided range to the piecewise function.
+ * Add the linear function on the provided range to the piecewise function.
```

Comment:
Add a given linear function on the provided range of the piecewise function.

Reply:
Done.